lib/util/path-util.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @fileoverview Common helpers for operations on filenames and paths
+ * @author Ian VanSchooten
+ * @copyright 2016 Ian VanSchooten. All rights reserved.
+ * See LICENSE in root directory for full license.
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var path = require("path"),
+    isAbsolute = require("path-is-absolute");
+
+//------------------------------------------------------------------------------
+// Private
+//------------------------------------------------------------------------------
+
+/**
+ * Replace Windows with Unix style paths and remove ./ prefix and any
+ * unnecessary "../"
+ *
+ * @param {string} filepath Path to normalize
+ * @returns {string} Normalized filepath
+ */
+function normalizeFilepath(filepath) {
```

Comment:
Yes.  I kept the same name it had previously but this is doing more than just normalizing.  

I believe the latest version of node-ignore allows Windows paths now (since this morning).  I'm going to check if that's the case, and if so attempt to avoid converting to linux paths, in which case the name could stay as-is.